cmd/feed-ingress/main.go

```diff
@@ -41,7 +47,13 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 
-	lb := ingress.NewLB()
+	lb := ingress.NewLB("/usr/sbin/nginx", nginxConfDir, nginxPort, &ingress.DefaultSignaller{})
+	err := lb.Start()
+	if err != nil {
+		log.Error("Unable to start nginx", err)
+		os.Exit(-1)
+	}
+
```

Comment:
It doesn't look like `lb.Stop()` is ever called. It should happen when controller is stopped.